src/mysql_to_sqlite3/transporter.py

```
@@ -650,6 +654,127 @@ def _create_table(self, table_name: str, attempting_reconnect: bool = False) ->
+                        definition = m.group(1).strip().rstrip(";")
+                    else:
+                        # As a last resort, try to use the full statement replacing the prefix
+                        # Not ideal, but better than failing outright
+                        idx = create_stmt_str.upper().find(" AS ")
+                        if idx != -1:
+                            definition = create_stmt_str[idx + 4 :].strip().rstrip(";")
+            except mysql.connector.Error:
+                pass
+
+        if not definition:
+            raise sqlite3.Error(f"Unable to fetch definition for MySQL view '{view_name}'")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)